app/models/project.rb

```diff
@@ -6,6 +6,7 @@ class Project < ActiveRecord::Base
 
   has_many :first_choice_application_drafts,  class_name: 'ApplicationDraft', foreign_key: 'project1_id'
   has_many :second_choice_application_drafts, class_name: 'ApplicationDraft', foreign_key: 'project2_id'
+  has_many :accepted_teams, class_name: 'Team' # working on the project; after application procedure
```

Comment:
I'd vote for `accepted` over `participating`. 
New bid: `assigned_teams`. That doesn't cover the 'legal' relationship, but it communicates the bond between team and project. Okay?